ovos_utils/gui.py

```diff
@@ -606,28 +616,35 @@ def setup_default_handlers(self):
 
     def upload_gui_pages(self, message: Message):
         """
-        Emit a response Message with all known GUI resources managed by
-        this interface.
+        Emit a response Message with all known GUI files managed by
+        this interface for the requested infrastructure
         @param message: `gui.request_page_upload` Message requesting pages
         """
         if not self.ui_directories:
             LOG.debug("No UI resources to upload")
             return
         request_res_type = message.data.get("framework", "qt5")
```

Comment:
Mostly chose this one because all existing skills that I know of will only have a `ui` drectory. `all` is intended to point at `{skill_root}/gui` or an equivalent directory containing per-framework directories

Reply:
I guess I could do this differently and default to `all` if the directory exists, else `qt5`. That would go better with the workshop changes